scripts/post-install/post-install.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const Logger = require('../utils/logger');
+
+fs.readFile(`${process.env.INIT_CWD}/package.json`, (error, file) => {
```

Comment:
what is the difference between `process.env.INIT_CWD` and `process.cwd()`?

Reply:
This is a tricky thing with npm. `process.cwd()` doesn't return the initial process directory, it returns the package being installed as the current working directory. `INIT_CWD` was added as a work around for this issue. I found a short discussion about it here: 

https://github.com/npm/npm/issues/16990

Reply:
Might be good to leave a code comment about this and link to the npm issue.

Reply:
Added a short commend and reference: https://github.com/cerner/terra-toolkit/pull/326/commits/6b9636e5d7d61cd6afeab5b9fc973e7b12b1fb03